android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/ConfigurationRegistry.kt

```diff
@@ -696,7 +696,8 @@ constructor(
                       TYPE_REFERENCE_DELIMITER,
                     )
                   val resourceId = listEntryComponent.item.reference.extractLogicalIdUuid()
-                  val listResourceUrlPath = "$resourceKey?$ID=$resourceId&_count=$DEFAULT_COUNT"
+                  val listResourceUrlPath =
+                    "${ResourceType.List.name}?$resourceKey?$ID=$resourceId&_page&_count"
```

Comment:
@Raynafs we might have to add a default page and count if someone hasn't provided them

Reply:
Oh yes sure. Adding it